src/harness/certs/generate_fake_certs.sh

```diff
@@ -128,19 +125,19 @@ echo "\n\nGenerate 'certs for devices' certificate/key"
 openssl req -new -newkey rsa:2048 -nodes \
     -reqexts cbsd_req -config ../../../cert/openssl.cnf \
     -out device_a.csr -keyout device_a.key \
-    -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=device_a"
+    -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=test_fcc_id_a:test_serial_number_a"
 openssl ca -cert cbsd_ca.cert -keyfile private/cbsd_ca.key -in device_a.csr \
     -out device_a.cert -outdir ./root \
-    -policy policy_anything -extensions cbsd_req_sign -config ../../../cert/openssl.cnf \
+    -policy policy_anything -extensions cbsd_req_sign_without_test_ext -config ../../../cert/openssl.cnf \
```

Comment:
Could you elaborate on why you don't use the TEST extension for this 2 certificates?

Reply:
from my understanding when the "TEST" extension presents in a certificate, the SAS should not check the fields (subject, CN, OU, FRNs , FCC Id & Cbsd serial number, role, or Any other applicable fields of specific certificate), it only needs to check that the certificate is valid and to ckeck its chain of trust.

so we need this TEST extension in the certificates that we will use in all the tests which are not related to security/certificate.

I deleted this extension of those certificates, because they are used in SIQ.7 and GRA.4  tests.
SIQ.7 and GRA.4 should use Cbsds' certificates which contain a correct subject( CN = fccId : Cbsd serial number) and the SAS will use this to identify that the request is sent by the appropriate  CBSD.

we can find important and useful details in the document WINNF-TS-0122  CBRS CBSD Test Spec (Appendix 9.1 Test Certificate Generation & Use)

please let us know your opinion about that.